tokio-trace/src/span.rs

```diff
@@ -467,12 +467,9 @@ impl Span {
     ///
     /// If this span is disabled, or the resulting follows-from relationship
     /// would be invalid, this function will do nothing.
-    pub fn follows_from<I>(&self, from: I) -> &Self
-    where
-        I: AsId,
-    {
+    pub fn follows_from<I>(&self, from: impl for<'a> Into<Option<&'a Id>>) -> &Self {
```

Comment:
Am I going crazy or is `I` used anywhere? seems like it can be removed since you are using impl  Trait.

Reply:
Nope, it's totally unused, I just forgot to get rid of it. My bad, thanks for noticing!